tests/integration/telemetry/CLAUDE.md

```
@@ -0,0 +1,12 @@
+# Tests: integration/telemetry
+
+## Available fixtures (this directory's conftest.py)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)